docs/tech-spike-write-ups/get-an-identity-oidc-oauth-with-ruby.md

```diff
@@ -0,0 +1,126 @@
+# Get an identity spike (Ruby) - Risks identified
+Author: Theodor Vararu
+Date: 2022/07/18
+## Summary
+While the spike to build “Get an identity” as a Single Sign-on (SSO) provider based on OAuth v2.0 and OpenID Connect was successful, it uncovered a number of significant risks to taking this approach.
+
+This document outlines these risks, and presents some alternative approaches.
+## Background
+As part of making it easier for teachers to sign into DFE services, and separating their identity from metadata like Teacher Reference Numbers (TRNs), we hypothesised a system that would allow users to authenticate using SSO.
```

Comment:
```suggestion
This document outlines these risks, and presents some alternative approaches.

## Background

As part of making it easier for teachers to sign into DFE services, and separating their identity from metadata like Teacher Reference Numbers (TRNs), we hypothesised a system that would allow users to authenticate using SSO.
```